### ISSUE TITLE
Doesnt seem to be working

### ISSUE THREAD

**Issue by ghost** [Arkshine/AdminFreeLook#14]
The **new pre-realease version 1.5.5** doesnt seems to be working.
I installed everything correctly.
It shows up on 'meta list'.

This is my cvars (on server.cfg): 

> afl_enabled 1
> afl_admin_access_flags k
> afl_user_override_mode d

There is no server/error logs about AdminFreeLook.
Last version 1.5.4 works.
Also, iam using amx 1.8.2.

**Comment by Arkshine** [comment]
You should have output in your server log (not AMXX). Typically the same log where you see server infos and metamod/amxx loading, like:

```
Console initialized.
Protocol version 48
Exe version 1.1.2.7 (cstrike)
Exe build: 13:13:28 Aug 29 2013 (6153)
STEAM Auth Server
Server IP address 192.168.0.2:27016

   Metamod version 1.21.1-am  Copyright (c) 2001-2013 Will Day <willday@metamod.org>
   Metamod comes with ABSOLUTELY NO WARRANTY; for details type `meta gpl'.
   This is free software, and you are welcome to redistribute it
   under certain conditions; type `meta gpl' for details.


   AMX Mod X version 1.8.3-dev+5033 Copyright (c) 2004-2015 AMX Mod X Development Team
   AMX Mod X comes with ABSOLUTELY NO WARRANTY; for details type `amxx gpl'.
   This is free software and you are welcome to redistribute it under
   certain conditions; type 'amxx gpl' for details.


   Admin Free Look v1.5.5-pre Copyright (c) 2015 Arkshine.
   Status: ready
```

**Comment by ghost** [comment]
I dont have any logs in my logs folder.

**Comment by Arkshine** [comment]
Well, if the module is loaded (`amxx modules`) and nothing happens, this likely it did not find the symbol/signatures in the mod.

Are using ReGameDLL? I actually did not try to look at it, but I'm willing to take a look. 

EDIT: Did not see, you're saying it was working in 1.5.4?  To answer about 1.5.5, it's just about fixing #10, refactoring my code a little, renaming cvars and such. Game data are unchanged. You have to stop/start your server to update.

**Comment by ghost** [comment]
Yes i have been using 1.5.4 for months now, it has small bugs but works well.
Iam not sure what you mean with ReGameDLL / stop/start update my server.

EDIT: No iam not using ReGameDLL, i tryed stop/start/restart plenty of times and its always the same. The module is running but doesnt work somehow.

**Comment by Syturio** [comment]
.

**Comment by Arkshine** [comment]
What are the values of mp_force\* cvars btw? `cvarlist mp_force`
